startvs.cmd

```diff
@@ -27,4 +27,4 @@ IF NOT EXIST "%DOTNET_ROOT%\dotnet.exe" (
     exit /b 1
 )
 
-start %sln%
+start "" "%sln%"
```

Comment:
Quotes surrounding solution make sense but the extra argument to `start` does not.

Reply:
according to [docs](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/start#syntax) the first double quotes goes for the Title